### DISCUSSION TITLE
Error reconciling KafkaTopic null

### DISCUSSION THREAD

**Discussion by obbiondo** [strimzi/strimzi-kafka-operator#8137] [Q&A]
Hello, I'm encountering a strange error in the TopicOperator. I've cloned a cluster with MirrorMaker2, and some topics were created automatically, and then they became KafkaTopic with the TO topic synchronization.
The problem is that TO reconciliation continuously tries to create again the topic, and since it exists it returns an error.

I've attached TopicOperator logs, Kafka and some KafkaTopic resources.
Do you have any ideas?

[log_TO.txt](https://github.com/strimzi/strimzi-kafka-operator/files/10793748/log_TO.txt)
[kafka.txt](https://github.com/strimzi/strimzi-kafka-operator/files/10793784/kafka.txt)
[kt1.txt](https://github.com/strimzi/strimzi-kafka-operator/files/10793794/kt1.txt)
[kt2.txt](https://github.com/strimzi/strimzi-kafka-operator/files/10793795/kt2.txt)

**Comment by scholzj** [comment]
Have you tried to restart the Topic Operator pod? Did it helped?
